shell/app/modules/project/common/components/workflow/steps/base-step.tsx

```diff
@@ -15,18 +15,25 @@ import React from 'react';
 
 export interface IBaseProps {
   enable?: boolean;
+  beta?: boolean;
 }
 
 const BaseStep: React.FC<{
   className?: string;
   title: string;
   extra?: React.ReactNode;
-}> = ({ className, title, children, extra }) => {
+  beta?: boolean;
+}> = ({ className, title, children, extra, beta }) => {
   return (
     <div className={`workflow-step bg-white rounded-sm w-[280px] shadow-card flex-shrink-0 mx-2 ${className}`}>
       <div className="px-2 rounded-t-sm bg-default-02 h-8 flex items-center justify-between mb-0">
         <span className="font-medium">{title}</span>
-        <div>{extra}</div>
+        <div>
+          {extra}
+          {beta ? (
+            <span className="ml-2 px-2 rounded-full text-xs text-danger border border-danger border-solid">beta</span>
+          ) : null}
+        </div>
```

Comment:
Why not just add beta flag here instead of pass from props? I assume this flag will be removed at the same time in the future?